docs/writing-stories/args.md

```diff
@@ -0,0 +1,127 @@
+---
+title: 'Args'
+---
+
+A story is a component with a set of arguments (props, slots, inputs, etc). “Args” are Storybook’s mechanism for defining those arguments as a first class entity that’s machine readable. This allows Storybook and its addons to live edit components. You do not need to change your underlying component code to use args.
+
+When an arg’s value is changed, the component re-renders, allowing you to interact with components in Storybook’s UI via addons that affect args.
+
+
+For a discussion on how and why to write stories with args, check the [using args](./introduction#using-args) section. For details on how args work, read on.
+
+
+### Args object
+
+The args object can be defined at the story and component level (see below). It is an object with string keys, where values can have any type that is allowed to be passed into a component in your framework.
+
+### Story args
+
+To define the args of a single story, use the `args` CSF story key:
+
+```js
+export const Primary = Story.bind({});
+Primary.args = {
+  primary: true,
+  label: ‘Primary’,
+}
+```
+
+<div style="background-color:#F8FAFC">
+TODO: ask dom/tom for clarity on the reuse link location
```

Comment:
I think I was thinking of this: https://docs.google.com/document/d/1BawdfqGhsP5tqhYQbVn29c6_rAz1ePmk9QayoXLDOKM/edit#heading=h.mjsq7rpmc8oq

Reply:
the link was updated per feedback